code/modules/clothing/head/helmet.dm

```diff
@@ -154,6 +154,35 @@ obj/item/clothing/head/blob
 	flags = HEADCOVERSEYES|HEADCOVERSMOUTH
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES
 
+/obj/item/clothing/head/helmet/knight
+	name = "medieval helmet"
+	desc = "A classic metal helmet."
+	icon_state = "knight_green"
+	item_state = "knight_green"
+	armor = list(melee = 41, bullet = 15, laser = 5,energy = 5, bomb = 5, bio = 2, rad = 0)

```

Comment:
Some of the values here are oddly specific, and they don't match up with the armor oddly enough.